matRad/optimization/projections/matRad_LETdProjection.m

```diff
@@ -0,0 +1,96 @@
+classdef matRad_LETdProjection < matRad_BackProjection
```

Comment:
The LETdProjection could probably derive from the LETDose projection (and maybe the pyhsicalDose projection) and then just call the superclass methods to obtain their intermediate results?)